### PR TITLE
Makes MoMMI jetpacks usable

### LINKED ISSUE #16266: Mommis are missing action buttons for the carbon jetpack.
Like the title reads mommis can take the jetpack upgrade but get no action buttons or verbs to toggle it on etc.

### PR DESCRIPTION
**Pull request by Pathid**
Closes #16266
:cl:
* bugfix: Once installed, robot jetpacks now function in MoMMIs as they were supposed to. Godspeed, little crabs.

### CODE REVIEW

File: code/modules/mob/living/silicon/mommi/mommi_movement.dm [hunk 1]
```diff
@@ -2,11 +2,14 @@
 /mob/living/silicon/robot/mommi/Process_Spaceslipping(var/prob_slip=5)
 	return 0
 
-/mob/living/silicon/robot/mommi/Process_Spacemove()
+/mob/living/silicon/robot/mommi/Process_Spacemove(var/check_drift = 0)
```
Comment (DamianX) [inline code/modules/mob/living/silicon/mommi/mommi_movement.dm]: Isn't this proc equal to the parent? Am I missing something?

**Comment by DamianX** [comment]
Upon testing, the action buttons are greyed out, might want look into that.

**Comment by Pathid** [comment]
That's odd. My testing didn't show any problems. The buttons took about a second to become active.

Putting the Grant proc there worked surprisingly well on my end. There's no way to get the MMI out without destroying the MoMMI mob (which gets rid of the buttons) and reset boards are already rejected by MoMMIs so I couldn't find a normal way for the buttons to show up when they shouldn't.

File: code/modules/mob/living/silicon/mommi/mommi_movement.dm (hunk 1 above)
Comment (Pathid) [reply code/modules/mob/living/silicon/mommi/mommi_movement.dm]: Yeah I misread something. Changing it.

File: code/modules/mob/living/silicon/mommi/mommi_movement.dm [hunk 2]
```diff
@@ -1,13 +1,13 @@
 // MoMMIs do spess construction, so shouldn't slip.
 /mob/living/silicon/robot/mommi/Process_Spaceslipping(var/prob_slip=5)
 	return 0
-
-/mob/living/silicon/robot/mommi/Process_Spacemove()
+/*
```
Comment (DamianX) [inline code/modules/mob/living/silicon/mommi/mommi_movement.dm]: just remove this

**Comment by DamianX** [comment]
Still grey
image

**Comment by Pathid** [comment]
Maybe you should walk me though what you did to get that.

mommijet

**Comment by Pathid** [comment]
Nevermind, I think I figured it out. Now to fix it somehow.

**Comment by DamianX** [comment]
Need help with this? What's the reason my buttons are grey?

**Comment by Pathid** [comment]
See, I was putting the upgrade board in as the MoMMI itself. If you put it in a player-less MoMMI and then take control of it with admin tools it seems to break the buttons.

**Comment by DamianX** [comment]
I'm going to blame action buttons being weird and merge this, it should work fine in regular gameplay.

**Pull request merged**